src/Accelerators/NNPA/src/CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+
+# add_dlc_library(name sources...
```

Comment:
This looks like a copy of the `add_library` function from MLIR.cmake. Is the intention to add a new global library set?

Reply:
Yeah, this is a mess .... I will tidy up in a follow-on PR if that's ok?

Reply:
I think it would be better to do it now or in a separate PR that goes in before this change since things that get postponed often get postponed indefinitely.

Also just noticed that there's a second `src` directory under the accelerators path. Is it needed? It seems redundant.

Reply:
removed the extra src
still working on the add_dlc_library